### PR TITLE
Setup Navbar

### PR DESCRIPTION
**Pull request by bharget**
This by no means needs to stay like this but I figured I would go ahead and get it setup on master.

### CODE REVIEW

**Comment by bharget** [comment]
I figure once we have overrides for bulma, we can put those in a file called `overrides.sass` or something.

File: app/frontend/src/javascripts/navbar.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(
+    document.querySelectorAll(".navbar-burger"),
+    0
+  );
+
+  // Check if there are any navbar burgers
+  if ($navbarBurgers.length > 0) {
+    // Add a click event on each of them
+    $navbarBurgers.forEach(el => {
```
Comment (jhurd2326) [inline app/frontend/src/javascripts/navbar.js]: Are there cases where there would be more than one navbar burger?

File: app/frontend/src/stylesheets/custom.sass [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// Use this file to place custom classes not included with Bulma
```
Comment (jhurd2326) [inline app/frontend/src/stylesheets/custom.sass]: Hopefully we won't have to use this a ton but if for some reason we do end up using this a bit then we may want to consider breaking our custom styles out into separate files but we can tackle that if/when we get there.

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -5,4 +5,9 @@ def turbolinks_cache_control_meta_tag
     cache = turbolinks_cache_control
     content_tag(:meta, nil, name: "turbolinks-cache-control", content: cache) if cache.present?
   end
+
+  def static_navbar?
```
Comment (jhurd2326) [inline app/helpers/application_helper.rb]: I would lean towards not doing this check if it's not needed right now and just render the navbar all the time. If for some reason we don't need to render the navbar or want to render something differently down the road, I vote we just change it then.

**Pull request merged**